sdk/iOS/ZumoE2ETestApp/ZumoE2ETestApp/ZumoAppDelegate.m

```diff
@@ -12,6 +12,10 @@ @implementation ZumoAppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
     if([application respondsToSelector:@selector(registerForRemoteNotifications)]) {
+        
+        UIUserNotificationSettings *notificationSettings = [UIUserNotificationSettings settingsForTypes:UIUserNotificationTypeAlert | UIUserNotificationTypeBadge | UIUserNotificationTypeSound categories:nil];
+        [application registerUserNotificationSettings:notificationSettings];
+        
         [application registerForRemoteNotifications];
```

Comment:
This is as-is from the documentation at https://azure.microsoft.com/en-us/documentation/articles/mobile-engagement-ios-get-started/